### ISSUE TITLE
Incremental loading (Postgres/Redshift) fails when using datetime

### ISSUE THREAD

**Issue by cosmok** [embulk/embulk-input-jdbc#167]
The parameterized values   are double quoted which will not work in Postgres/Redshift

eg,
```
query: |
     SELECT * FROM my_table WHERE
      SessionUpdateTime >= :SessionUpdateTime ORDER BY SessionUpdateTime ASC;
  use_raw_query_with_incremental: true
  incremental_columns: [SessionUpdateTime]
  incremental: true
  last_record: ["{{ env.start_date_time }}"] <-- need to quote as otherwise yaml parser complains on date-times like these: 2019-08-08 12:00:00
```
sql ends up like this:

```
 SELECT * FROM my_table WHERE
      SessionUpdateTime >=  "2019-08-08 12:00:00" ORDER BY SessionUpdateTime ASC
```

which errors out (due to the double quote), ideally it should be:

```
SELECT * FROM my_table WHERE
      SessionUpdateTime >=  '2019-08-08 12:00:00' ORDER BY SessionUpdateTime ASC
```
```

**Comment by hito4t** [comment]
@cosmok 
Would you offer the following information?
- DDL for `my_table `
- error message when `env.start_date_time` is not quoted
- error message when `env.start_date_time` is double quoted

**Comment by hito4t** [comment]
@cosmok 
I could reproduce the issue.

When I didn't quote `{{ env.start_date_time }}`, I got following error.

<pre>
Error: while scanning a plain scalar
 in 'string', line 15, column 17:
      last_record: [2019-08-21 11:00:00]
                    ^
found unexpected ':'
 in 'string', line 15, column 30:
      last_record: [2019-08-21 11:00:00]
                                 ^
</pre>

When I quoted `{{ env.start_date_time }}` by `""`, I got following error.

<pre>
Error: org.embulk.config.ConfigException: Invalid timestamp without time zone pattern: "2019-08-21 11:00:00"
</pre>

This is because the format of `env.start_date_time` is improper.

If the type for the `SessionUpdateTime` column is `timestamp`, `env.start_date_time` should be like `2019-08-21T11:00:00.000000`.
If the type for the `SessionUpdateTime` column is `timestamp with time zone`, `env.start_date_time` should be like `2019-08-21T11:00:00.000000+0300`.

Neither README nor the error message say the correct format.
I'll revise them.
